sellotape/sellotape_dj/main_app/models.py

```diff
@@ -4,9 +4,10 @@
 class Stream(models.Model):
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     link = models.URLField(max_length=500, blank=False)
-    airs_on = models.DateField()
-    ends_on = models.DateField(null=True)
-    added_on = models.DateField()
+    title = models.CharField(max_length=50, default='')
```

Comment:
Note that I also added a description field, (#37)
I think it can contribute to the content of the site.